### ISSUE TITLE
[Screen Reader-Social Embedded-Social Activities]: TalkBack does not announce the name for "More option" button.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#17]
"Priority: High

Repro Steps
1. Launch the One Bus Away App.
2. Navigate to ""Sign In"" link under Hamburger link and press ENTER.
3. Sign in screen showing all currently supported identity service providers will appear.
4. Use any service provider for log in.
5. Select any location bubble and see the given arrivals.
6. Navigate to ""Join Discussion"" button and press ENTER.
7. Navigate to ""More option"" (ellipses) button.
8. Verify the speech of talkback for ""More option"" button.

Note:
1. Same issue is repro for ""Like"", ""Comment"", ""Pin"", ""Take a picture"" buttons and after uploading picture, the uploaded image button also. (see attachment)

Expected
TalkBack does not announce the name for ""More option"" button.Also talkback should announce the state as ""selected"" for the selected buttons like- like, pin etc.

Actual
Talback should announce the name for ""More Option"" button. Also talkback does not announce the state as ""selected"" for the selected buttons like- like, pin etc.

MAS Rule: MAS4.2.1

Impact
If relevant names are not defined for the controls on the page, then screen reader users will not be able to understand the purpose of using different controls on the page.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back"
